cypress/integration/transfer/reorder-with-buttons.feature

```diff
@@ -0,0 +1,41 @@
+Feature: Reorder items in the selected list
+
+  Background:
+    Given reordering of items is enabled
+
+# I created two scenarios for up and down reordering because I wanted to incorporate a check for 'is first' or 'is last' and it seemed overly ambiguous to write this in a single scenario. Or, is it enough that I have included the 'Disable move buttons' scenarios at the bottom of this file, so that doesn't need to be handled in the basic scenarios at the top?
+
+  Scenario Outline: Reorder items upwards using buttons
+      Given a selected list that has two or more items
+      And the <previous> item is highlighted
+      When the 'move up' button is pressed
+      Then the highlighted item is moved to the <next> place
+
+      Examples:
+          | previous | next |
+          | 1        | 1    |
+          | 2        | 1    |
+          | 3        | 2    |
+
+  Scenario Outline: Reorder items upwards using buttons
+      Given a selected list that has two or more items
+      And the <previous> item is highlighted
+      When the 'move down' button is pressed
+      Then the highlighted item is moved to the <next> place
+
+      Examples:
+          | previous | next |
+          | 1        | 2    |
+          | 2        | 3    |
+          | 3        | 3    |
```

Comment:
In this case the first `Given` needs to be specific: `Given the selected list has three items`